app/templates/user/manage.html

```diff
@@ -1,48 +1,158 @@
+                                   value="{{ GET.username }}">
+                        </div>
+                        <br>
+                        <div class="input-group">
+                            <div class="input-group-prepend">
+                                <label class="input-group-text" for="filter-first_name">Vorname:</label>
+                            </div>
+                            <input type="hidden" id="filter-first_name-type" name="first_name_type" value="contain">
+                            <input class="form-control" type="text" name="first_name" id="filter-first_name"
+                                   value="{{ GET.first_name }}">
+                        </div>
+                        <br>
+                        <div class="input-group">
+                            <div class="input-group-prepend">
+                                <label class="input-group-text" for="filter-last_name">Nachname:</label>
+                            </div>
+                            <input type="hidden" id="filter-last_name-type" name="last_name_type" value="contain">
+                            <input class="form-control" type="text" name="last_name" id="filter-last_name"
+                                   value="{{ GET.last_name }}">
+                        </div>
+                        <br>
+                        <div class="input-group">
+                            <div class="input-group-prepend">
+                                <label class="input-group-text" for="filter-email">E-Mail:</label>
+                            </div>
+                            <input type="hidden" id="filter-email-type" name="email_type" value="contain">
+                            <input class="form-control" type="text" name="email" id="filter-email"
+                                   value="{{ GET.email }}">
+                        </div>
+                        <br>
+
+                        <div class="dropdown">
+                            <div class="input-group">
+                                <div class="input-group-prepend">
+                                    <label class="input-group-text" for="filter-date_joined">Registriert:</label>
+                                </div>
+                                <input class="form-control" type="date" name="date_joined_start"
+                                       value="{{ GET.date_joined_start }}">
+                                <div type="text" class="form-control form-control-text">bis</div>
+                                <input class="form-control" type="date" name="date_joined_end"
+                                       value="{{ GET.date_joined_end }}">
+                            </div>
+                            <datalist id="filter-owner-data-list"></datalist>
+                        </div>
+                        <br>
+                        <div class="dropdown">
```

Comment:
Somehow, Dark Mode doesn't work here (date filter)
-> same in app/templates/ticket/manage.html